### DISCUSSION TITLE
Developer Profile under review for months!!

### DISCUSSION THREAD

**Discussion by as2621-dev** [amzn/selling-partner-api-models#3344] [General]
Hello, 

I have been struggling to get a developer profile approved for so long now. First, I created a public developer profile case ID7334603642. That profile has been under review for months now, I have not heard back from Amazon regarding any additional information. 
Then I figured it would be better if I can just create a Private developer profile but since my Public developer profile is under review I cant get a private developer profile self-authentication done. 

What should I do? can someone please help me? I have been sending reminders to Amazon but no response.

**Comment by as2621-dev** [comment]
Can someone please help?

**Reply by AtikSrl** [reply]
Hello @ashesh2621 
I’m also in the same situation as you but for less time... and I still do not receive answers or guidance on this
reading various posts I found that there are many to complain about this problem

Have you tried other channels than this? have you had any feedback?
